book/website/collaboration/research-infrastructure-roles/data-wrangler.md

```diff
@@ -0,0 +1,84 @@
+(cl-infrastructure-data-wranglers)=
+# Data Wranglers: Overview
+
+Data Wranglers can be viewed as a specialised type of Data Scientist, primarily working in the space between data generators and data analysts.
+The terminology used to describe various data science activities (for example data collection, wrangling, analysis, modelling, visualisation), and how they map onto different job titles, is domain specific and will vary on a project and organisational level.
+
+(cl-infrastructure-data-wranglers-tasks)=
+## What do Data Wranglers do?
+
+In a data science project, it is commonly observed that data wrangling tasks take the majority of time {cite:ps}`anaconda2020sods`, in contrast to data analysis and modelling.
+Traditionally, data wrangling tasks involve cleaning, restructuring and filtering data into standardised research ready formats.
+However, in terms of Data Wrangler as a profession, day to day tasks and objectives can be much more diverse.   
+
+Data Wranglers work primarily in the space between data generators and data analysts, who are addressing the research question of interest.
+Understanding the intended use for the data in the context of the analysis and research questions, Data Wranglers can be in the position of influencing data generators in improving data collection methods, ultimately benefiting all points of the data life cycle.
+Similarly, Data Wranglers will conduct preliminary analysis on the data to ensure both completeness of data and preparation for data analysis, acting as a proxy for the data generator’s knowledge during the data analysis process.
+As a key focus of a Data Wrangler’s role is the preparation of research ready data {cite:ps}`stewart2022table`, data security, data management and FAIR standards are core to the role {cite:ps}`mons2017fair`.
```

Comment:
This line reads a little confusing to me. Perhaps:
> A key focus of a Data Wrangler’s role is the preparation of research ready data {cite:ps}`stewart2022table` in which data security, data management and FAIR standards are also core responsibilities {cite:ps}`mons2017fair`

Reply:
Much better, thanks! But maybe "responsibilities" can be replaced by "priorities" and then it sounds even better?

Reply:
> A key focus of a Data Wrangler’s role is the preparation of research ready data {cite:ps}stewart2022table in which data security, data management and FAIR standards are all core priorities {cite:ps}mons2017fair

Reply:
purrfect 🐈